internal/guest/runtime/runc/runc.go

```diff
@@ -53,13 +53,12 @@ func (r *runcRuntime) initialize() error {
 	for _, p := range paths {
 		_, err := os.Stat(p)
```

Comment:
This was already here obviously but I wonder if it's ill form to just MkdirAll and skip the Stat, as MkdirAll stats internally anyways to decide whether to just no-op. The case where the directory doesn't exist would end up stating twice

Reply:
So squash it even further like:
```go 
for _, p := range paths {
	if err := os.MkdirAll(p, 0700); err != nil {
		return errors.Wrapf(err, "failed making runC container files directory %s", p)
	}
}
```